### ISSUE TITLE
Method call arg indentation is off

### ISSUE THREAD

**Issue by marcusklaas** [rust-lang-nursery/rustfmt#1035]
This is currently in visitor.rs:

``` rust
    let rewrite = e.rewrite(&self.get_context(),
                 self.config.max_width - self.block_indent.width(),
                 self.block_indent)
        .unwrap_or_else(|| self.snippet(e.span));
```

**Comment by marcusklaas** [comment]
The arguments of `rewrite` are being indented as if the call was indented on block level.

**Comment by imjacobclark** [comment]
PR open - #1043

**Comment by topecongiro** [comment]
The above code does not exist in visitor.rs anymore, but the problem itself is fixed in `0.8.3 ()`.
